We need to insert the sysroot path into compiler commandlines for this
kind of variable use, otherwise we're looking into /usr/include/.

RP - 26/11/09

Index: git/xorg-server.m4
===================================================================
--- git.orig/xorg-server.m4	2009-11-26 23:11:03.000000000 +0000
+++ git/xorg-server.m4	2009-11-26 23:12:28.000000000 +0000
@@ -30,7 +30,7 @@
 
 AC_DEFUN([XORG_DRIVER_CHECK_EXT],[
 	SAVE_CFLAGS="$CFLAGS"
-	CFLAGS="$CFLAGS -I`pkg-config --variable=sdkdir xorg-server`"
+	CFLAGS="$CFLAGS -I$PKG_CONFIG_SYSROOT_DIR`pkg-config --variable=sdkdir xorg-server`"
 	AC_COMPILE_IFELSE([AC_LANG_PROGRAM([[
 #include "xorg-server.h"
 #if !defined $1
